### PR TITLE
code & doxygen cleanup + readme + local image

### PR DESCRIPTION
**Pull request by hoffmannjan**

### CODE REVIEW

File: Adafruit_INA219.cpp [hunk 1]
```diff
@@ -102,8 +81,8 @@ void Adafruit_INA219::setCalibration_32V_2A(void)
   // setting the VBUS_MAX to 16V instead of 32V, etc.
 
   // VBUS_MAX = 32V             (Assumes 32V, can also be set to 16V)
-  // VSHUNT_MAX = 0.32          (Assumes Gain 8, 320mV, can also be 0.16, 0.08, 0.04)
-  // RSHUNT = 0.1               (Resistor value in ohms)
+  // VSHUNT_MAX = 0.32          (Assumes Gain 8, 320mV, can also be 0.16, 0.08,
+  // 0.04) RSHUNT = 0.1               (Resistor value in ohms)
```
Comment (siddacious) [inline Adafruit_INA219.cpp]: This is formatted a little confusingly; can you have the gain values continue on a line above?

File: Adafruit_INA219.h [hunk 2]
```diff
@@ -17,202 +17,120 @@
...
+/** values for operating mode **/
+enum {
+  INA219_CONFIG_MODE_POWERDOWN = (0x0000),
+  INA219_CONFIG_MODE_SVOLT_TRIGGERED = (0x0001),
+  INA219_CONFIG_MODE_BVOLT_TRIGGERED = (0x0002),
+  INA219_CONFIG_MODE_SANDBVOLT_TRIGGERED = (0x0003),
+  INA219_CONFIG_MODE_ADCOFF = (0x0004),
+  INA219_CONFIG_MODE_SVOLT_CONTINUOUS = (0x0005),
+  INA219_CONFIG_MODE_BVOLT_CONTINUOUS = (0x0006),
+  INA219_CONFIG_MODE_SANDBVOLT_CONTINUOUS = (0x0007),
```
Comment (siddacious) [inline Adafruit_INA219.h]: Since these start at zero and are contiguous, I think you can skip the assignment.

Comment (hoffmannjan) [reply Adafruit_INA219.h]: yeah, I changed It :)

File: Adafruit_INA219.cpp (hunk 1 above)
Comment (hoffmannjan) [reply Adafruit_INA219.cpp]: you are right, I'm using `clang-format` and It sometime do things like this :)

File: Adafruit_INA219.h (hunk 2 above)
Comment (hoffmannjan) [reply Adafruit_INA219.h]: tested It on UNO to measure battery and works fine after changes

**Comment by siddacious** [comment]
Perfect, thanks :)

**Pull request merged**